src/Subsystems/BallConveyor.cpp

```diff
@@ -0,0 +1,29 @@
+#include "BallConveyor.h"
+#include "../RobotMap.h"
+#include "CANTalon.h"
+
+const char BallConveyor::kSubsystemName[] = "BallConveyor";
+
+std::shared_ptr<BallConveyor> BallConveyor::self;
+
+std::shared_ptr<BallConveyor> BallConveyor::getInstance() {
+	if (! self) {
+	   self = std::make_shared<BallConveyor>();
+	}
+	return self;
+}
+
+BallConveyor::BallConveyor() : Subsystem(kSubsystemName),
+	kID1(RobotMap::kID1),
+	kID2(RobotMap::kID2)
+		{
```

Comment:
I have fixed the Tab problem on triple tape I do not know about the other computers.